docker-compose.yml

```
@@ -104,25 +98,13 @@ services:
       # Buckets for testing provider data imported from s3 are subdirectories under
       # /tests/s3-data/
       - ./catalog/tests/s3-data:/data:rw
+      - ./docker/minio/load_to_s3_entrypoint.sh:/opt/minio/s3_entrypoint.sh:ro
     # Loop through subdirectories mounted to the volume and load them to s3/minio.
     # This takes care of filesystem delays on some local dev environments that may make
     # minio miss files included directly in the minio volume.
     # More info here: https://stackoverflow.com/questions/72867045
-    # This does *not* allow for testing permissions issues that may come up in real AWS.
-    # And, if you remove files from /tests/s3-data, you will need to use `just down -v`
-    # and `just up` or `just recreate` to see the minio bucket without those files.
-    entrypoint: >
-      /bin/sh -c "
-      /usr/bin/mc config host add s3 http://s3:5000 $${AWS_ACCESS_KEY} $${AWS_SECRET_KEY};
-      cd /data;
-      for b in */ ; do
-        echo \"Loading bucket $$b\"
-        /usr/bin/mc mb --ignore-existing s3/$$b
-        /usr/bin/mc cp --r $$b s3/$$b
-        /usr/bin/mc ls s3/$$b;
-      done ;
-      exit 0;
-      "
+    
+    entrypoint: /opt/minio/load_to_s3_entrypoint.sh
```

This reference is incorrect - on L101 the file is being mounted into the container at `/opt/minio/s3_entroypoint.sh`, so this needs to also reference the same file:
```suggestion
    entrypoint: /opt/minio/s3_entrypoint.sh
```

---

This is a typo from my end, will fix this.